src/clean-core/to_string.hh

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <clean-core/string.hh>
+#include <clean-core/string_stream.hh>
```

Comment:
now that this header is dual-purpose we should only forward declare those two

Reply:
Done. Moved remaining definitions into the `to_string.cc`